pkg/api/authn_test.go

```diff
@@ -954,7 +954,7 @@ func TestCookiestoreCleanup(t *testing.T) {
 		err = os.Chtimes(sessionPath, changeTime, changeTime)
 		So(err, ShouldBeNil)
 
-		imgStore := local.NewImageStore(rootDir, false, false, log, metrics, nil, nil, nil)
+		imgStore := local.NewImageStore(rootDir, false, false, log, metrics, nil, nil, nil, nil)
```

Comment:
We really need to shorten the args list into a struct we can pass around. Not related to your PR thouugh.